### PR TITLE
Update to ZIO 1.0.0-RC19-2

### PR DESCRIPTION
**Pull request by jeremyrsmith**
- Update ZIO
- Update uzhttp
- Copypasta zio-interop-cats (until it can be removed), since it's no longer published for Scala 2.11
- Misc cleanup

### CODE REVIEW

File: polynote-server/src/main/scala/polynote/server/package.scala [hunk 1]
```diff
@@ -71,7 +71,9 @@ package object server {
         case _ => ZIO.none
       }.catchAll {
         err => ZStream.fromEffect(Logging.error(err)).drain
-      }.unNone.ensuring {
+      }.collect {
+        case Some(frame) => frame
```
Comment (jonathanindig) [inline polynote-server/src/main/scala/polynote/server/package.scala:75]: Is this better than `unNone`?

Comment (jeremyrsmith) [reply polynote-server/src/main/scala/polynote/server/package.scala:75]: No, but `unNone` seems to be gone, so...

Comment (jonathanindig) [reply polynote-server/src/main/scala/polynote/server/package.scala:75]: 😮

Comment (jonathanindig) [reply polynote-server/src/main/scala/polynote/server/package.scala:75]: 🤨

**Comment by jeremyrsmith** [comment]
Closing because now RC20 is out... will open a PR to target that eventually

**Comment by jonathanindig** [comment]
😅

**Pull request closed without merging**